packages/evm/contracts/arbitration/IArbitrable.sol

```diff
@@ -16,37 +12,9 @@ contract IArbitrable is ERC165 {
     */
     event Ruled(IArbitrator indexed arbitrator, uint256 indexed disputeId, uint256 ruling);
 
-    /**
-    * @dev Emitted when new evidence is submitted for the IArbitrable instance's dispute
-    * @param arbitrator IArbitrator submitting the evidence for
-    * @param disputeId Identification number of the dispute receiving new evidence
-    * @param submitter Address of the account submitting the evidence
-    * @param evidence Data submitted for the evidence of the dispute
-    * @param finished Whether or not the submitter has finished submitting evidence
-    */
-    event EvidenceSubmitted(IArbitrator indexed arbitrator, uint256 indexed disputeId, address indexed submitter, bytes evidence, bool finished);
-
-    /**
-    * @dev Submit evidence for a dispute
-    * @param _disputeId Id of the dispute in the Court
-    * @param _evidence Data submitted for the evidence related to the dispute
-    * @param _finished Whether or not the submitter has finished submitting evidence
-    */
-    function submitEvidence(uint256 _disputeId, bytes calldata _evidence, bool _finished) external;
-
     /**
     * @dev Give a ruling for a certain dispute, the account calling it must have rights to rule on the contract
```

Comment:
By "rights to rule on the contract", we mean this IArbitrable instance, right?

So we could expect implementations to add their own authentication mechanism to protect this interface?

Reply:
> By "rights to rule on the contract", we mean this IArbitrable instance, right?

In the context of the arbitrator yes, but here it doesn't sound very well. Removed in d042bfc9e25617d667573d7efc824159c25502c9

> So we could expect implementations to add their own authentication mechanism to protect this interface?

Yes :)